inc/extras.php

```diff
@@ -69,3 +69,16 @@ function theme1_render_title() {
 	}
 	add_action( 'wp_head', 'theme1_render_title' );
 endif;
+
+function regala_create_social_icons() {
+	$titan = TitanFramework::getInstance( 'regala' );
+	
+	for ( $i = 0; $i <= 10; $i++ ) {
+		$url = $titan->getOption( 'social_' . $i );
+		if ( empty( $url ) ) {
+			continue;
+		}
+		
+		echo "<a href='{$url}' target='_blank'></a>";
```

Comment:
Instead of directly outputting `$url`, sanitize the output to `esc_attr( $url )`